File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.kt
class Solution {
    fun arrayStringsAreEqual(word1: Array<String>, word2: Array<String>): Boolean {
        val one = composeToString(word1)
        val two = composeToString(word2)
        return one == two
    }

    private fun composeToString(word: Array<String>): String {
        val sb = StringBuffer()
        for (i in word) {
            sb.append(i)
        }
        return sb.toString()
    }
}

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.cpp
class Solution {
public:
    bool arrayStringsAreEqual(vector<string>& word1, vector<string>& word2) {
        string one = "";
        for (auto &next : word1) {
            one += next;
        }

        string two = "";
        for (auto &next : word2) {
            two += next;
        }

        return one == two;
    }
};
